### DISCUSSION TITLE
Use `textStyles` in theme

### DISCUSSION THREAD

**Discussion by isBatak** [chakra-ui/chakra-ui#8878] [Ideas]
### Description

I would like to be able to use user defined `textStyles` in theme.

For example in `form-control.ts` theme:
```ts
const baseStyleHelperText = defineStyle({
  textStyle: 'caption',
})
```

Docs does not mention it anywhere
https://chakra-ui.com/docs/styled-system/text-and-layer-styles#text-styles

Codesandbox: https://codesandbox.io/p/sandbox/optimistic-yalow-g9y7qs?file=%2Ftheme.ts

### Problem Statement/Justification

Above example code doesn't apply `caption` textStyle

### Proposed Solution or API

Provided example code should work.

**Comment by veloware** [comment]
this has been covered a few times, e.g. https://github.com/chakra-ui/chakra-ui/issues/3197 and it's something that I run into very regularly.

I've just went with the suggestion in this comment - https://github.com/chakra-ui/chakra-ui/issues/3197#issuecomment-770355751 and has worked OK for me, but I agree, the API is somewhat confusing / misleading that textStyle quite often cannot be applied within another theme.

**Comment by segunadebayo** [comment]
`textStyles` are in the default theme. Define yours with `defineTextStyles`.
